migrations/versions/e08125f1e6bc_.py

```diff
@@ -0,0 +1,73 @@
+"""empty message
```

Comment:
The migration should be redone after the changes to the DB schema are integrated. The less DB migrations the easier it is to manage them. And as long as they are not deployed anywhere we can delete and redo them.